pkg/config/deployment.go

```diff
@@ -365,8 +365,16 @@ type WaitStageOptions struct {
 type WaitApprovalStageOptions struct {
 	// The maximum length of time to wait before giving up.
 	// Defaults to 6h.
-	Timeout   Duration `json:"timeout"`
-	Approvers []string `json:"approvers"`
+	Timeout        Duration `json:"timeout"`
+	Approvers      []string `json:"approvers"`
+	MinApproverNum int      `json:"minApproverNum" default:"1"`
+}
+
+func (w *WaitApprovalStageOptions) Validate() error {
```

Comment:
need to be called here https://github.com/pipe-cd/pipe/blob/0c3dce154f0425f7371cdabd47d2948cee7f4cf6/pkg/config/deployment.go#L102